neurokit2/epochs/eventrelated_utils.py

```diff
@@ -73,7 +73,7 @@ def _eventrelated_sanitizeoutput(data):
     return df
 
 
-def _eventrelated_rate(epoch, output={}, var="ECG_Rate"):
+def _eventrelated_rate(epoch, output={}, var="ECG_Rate", time_start=None, time_end=None):
```

Comment:
@DominiqueMakowski are time_start and time_end as argument names alright (intuitive?) - check out the documentation in the main functions

Reply:
maybe something more specific like `rate_start` and `rate_end`, that suggests that "it will only use the rate from this range"? and it would not scope other potential arguments?

Reply:
or `ratetime_start` and `ratetime_end`?

Reply:
hmm "ratetime" doesnt sound very sensible haha...how about rate_epoch_start? a bit long but it's consistent with epoch_start and it's epoch for rate (if it makes sense =))) )